plutarch-test/plutarch-test.cabal

```diff
@@ -94,17 +95,27 @@ common deps
 
 library
   import:          c, deps
-  hs-source-dirs:  src
+  hs-source-dirs:  plutarch-base
```

Comment:
This is kind of weird. I didn't notice how `plutarch-test`, the library, and the specs (the actual tests) are in the same directory. They ought to be separated IMO - but maybe for another PR.

Reply:
copied, I'll create an issue.

Reply:
actually I think I can just resolve it myself as the weirdness only arose from the changes I made, so I think it's fine if I include them in the PR?